index.html

```diff
@@ -112,27 +112,31 @@ <h5 class="cardtitle fw-bolder mx-2 text-center opacity-100">Network</h5>
       </div>
 
       <div class="card register-btn h-50 p-2 mx-auto d-flex flex-row justify-content-evenly align-items-center
-                rounded-0 mb-3">
-        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light text-decoration-none
+                rounded-0  my-4 ">
+        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light
           " href="#">Join Our Art Exhibition</a>
       </div>
       
     </section>
 
-    <section class="mt-4 mb-5 w-100 bg-light">
+    <section class="mt-4 mb-2 w-100 speaker-details">
       <div class="title d-flex flex-column justify-content-center">
         <h3 class="text-dark h4 fw-bold mt-4  text-center">Featured Speakers</h3>
         <hr class="hr mx-auto border mb-3 border-1 border-danger">
       </div>
 
       <!-- dynamic speakers data from app.js -->
-      <div class="speakers d-flex flex-column flex-md-row justify-content-between mx-md-auto px-md-5
-        mt-5 flex-md-wrap " id="speakers-section"></div>         
+      <!-- mobile version -->
+      <div class="speakers h-100 d-flex d-md-none flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-2 flex-md-wrap pt-4" id="speakers-section"></div>
+      <!-- desktop version -->
+      <div class="speakers d-none d-md-flex gap-1 flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-5 flex-md-wrap " id="desktop-speakers-section"></div>
     </section>
 
-    <div class="button">
-      <button class="cardtitle ms-5 fw-bold py-3 text-center my-4 border-light-subtle mx-auto ms-1
-                  text-dark opacity-100 text-light text-decoration-none rounded" id="show-more-btn">
+    <div class="button d-md-none w-100">
+      <button class="cardtitle mx-2 fw-bold py-3 text-center my-4 mx-auto ms-1
+        opacity-100 text-decoration-none rounded text-uppercase" id="show-more-btn">
         More <span><i class="ms-2 fa fa-chevron-down chevron-arrow"></i></span>
       </button>
     </div>
```

Comment:
- [ ] On the mobile view of your project, the `More Button` is still not properly centered on the page. As the previous reviewer mentioned, Kindly ensure that the `More Button` is well-centered so your design follows the layout of the original design :wink: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225451993-af9ef608-df73-4031-b795-9ea9dbea4d18.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/225452135-1e54740c-d095-4c72-bb64-991561484a7e.png)